tsconfig.json

```diff
@@ -21,6 +21,6 @@
       "~/*": ["./src/*"]
     }
   },
-  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx", ".next/types/**/*.ts"],
+  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx", ".next/types/**/*.ts", "src/hooks/useHeaderScroll.js"],
```

Comment:
I don't see any `useHeaderScroll` hook. Is this development leftover? If so, could you please remove the change. Thanks.

Reply:
modified . please look through it